airflow/serialization/schema.json

```diff
@@ -84,6 +84,7 @@
         "catchup": { "type": "boolean" },
         "is_subdag": { "type": "boolean" },
         "fileloc": { "type" : "string"},
+        "_source_code": { "type": "string" },
```

Comment:
This is not part of DAG. DAG is created by DAG file. We should store references to the file here instead of saving the source code of the file to avoid duplication.

Reply:
Yep, let's not serialize it